help/configure-who-can-unsubscribe-others.md

```diff
@@ -24,6 +24,9 @@ Organization administrators can unsubscribe other users from any channel.
 
```

Comment:
Missing period (unrelated to this PR):

1. Under **Channel permissions**, configure **Who can unsubscribe others from
   this channel**

Reply:
Added a commit for this.